RobotsAndPencils/Astro
# Add SwiftLint + update to latest Pods

## Summary
This updates the dependent Quick/Nimble pods to the latest versions + incorporates SwiftLint to the project. As part of that some non-functional code cleanup was done to clear the SwiftLint warnings.

Finally, there was a small KeychainAccess method signature change (lower cased first character in variable names to align with Swift standards). So we'll need bump the version up to 4.1.0 once this gets merged and we publish out to CocoaPods for distribution.

## How to test
1. Nothing to do but a review code and verify tests succeed
2. In addition, load up the project and run tests manually to verify there are no more Xcode warnings

## Areas of the app affected
- Dependent library updates (Quick/Nimble)
- Added SwiftLint code standard checking
